modules/services/cloud-bench/main.tf

```diff
@@ -0,0 +1,97 @@
+resource "sysdig_secure_cloud_account" "cloud_account" {
+  account_id = var.account_id
+  cloud_provider = "aws"
+  role_enabled = "true"
+}
+
+resource "aws_iam_role" "cloudbench_role" {
+  name = "SysdigCloudBenchRole"
+  assume_role_policy = data.aws_iam_policy_document.trust_relationship.json
+  tags = var.tags
+}
+
+data "sysdig_cloud_bench_user" "trusted_sysdig_role" {
+  sysdig_secure_endpoint = var.sysdig_secure_endpoint
+
+  // TODO this should come from the provider based on the secure URL.
+  arn = "arn:aws:iam::059797578166:user/noah.kraemer"
```

Comment:
Hardcoded for now, but the `arn` field will become an output of this new datasource that I will add to the provider.